Tests/Invoke-PxGet.Tests.ps1

```diff
@@ -0,0 +1,200 @@
+
+#Requires -Version 5.1
+Set-StrictMode -Version 'Latest'
+
+BeforeAll {
+    $script:testRoot  = $null
+    $script:moduleList = @()
+    $script:failed = $false
+    $script:testNum = 0
+
+    & (Join-Path -Path $PSScriptRoot -ChildPath 'Initialize-Test.ps1' -Resolve)
+
+    function Init
+    {
+        $script:testRoot = $null
+        $script:moduleList = @()
+        $script:failed = $false
+        $Global:Error.Clear()
+    }
+    
+    function GivenPxGetFile
+    {
+        param(
+            [Parameter(Mandatory)]
+            [string] $Contents
+        )
+
+        New-Item -Path $testRoot  -ItemType 'File' -Name "pxget.json" -Value $Contents
+    }
+
+    function New-TestRoot
+    {
+        $script:testRoot = Join-Path -Path $TestDrive -ChildPath ($script:testNum++)
+        New-Item -Path $script:testRoot -ItemType 'Directory'
+    }
+
+    function RemovePxGetFile

```

Comment:
This doesn't need to be its own function since it is only called in `Reset`. Move these two lines into `Reset` (which should get moved into `AfterEach`).